src/dex/dex.utils.cpp

```diff
@@ -82,4 +82,9 @@ namespace antara::mmbot
     {
         return ans.result == "success";
     }
+
+    std::vector<orders::order> to_orders(mm2::my_orders_answer answer)
```

Comment:
here the parameter of the function should be by reference

Reply:
Same for: `const orders::order to_order (mm2::trade_result res)`?

Reply:
Yep

Reply:
And you should take it as const ref i believe